ROADMAP.md

```diff
@@ -0,0 +1,67 @@
+# Katib 2019 Roadmap
+
+This document provides a high level view of where Katib will grow in 2019. These objectives are based off Katib's Critical User Journey (CUJ),
+which can be found [here](https://bit.ly/2QNKMwt).
+
+The original Katib design document can be found [here](https://docs.google.com/document/d/1ZEKhou4z1utFTOgjzhSsnvysJFNEJmygllgDCBnYvm8/edit#heading=h.7fzqir88ovr).
+
+# Katib 1.0 Readiness
+
+* Stabilize APIs for StudyJobs
+	* Beta by end of Q2, 1.0 by end of Q4
+	* Formalize naming conventions (we use different names like katib vs vizier in different places)
```

Comment:
refactor studyjob field name style #351 and renaming some confusing fields (saying requestCount and requestNumber)